types/pnpapi/tslint.json

```diff
@@ -1 +1,8 @@
-{ "extends": "dtslint/dt.json" }
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "no-single-declare-module": false,
```

Comment:
In lieu of @RyanCavanaugh 's feedback - you should be able to remove these lint rule adjustments now, right?